README.md

```diff
@@ -12,6 +12,11 @@ To monitor your source directory for changes, and recompile files if they change
 python build.py & python -m SimpleHTTPServer && fg
 ```
 
+For Python 3.6.0 : 
+```
+python build.py & python -m http.server && fg
```

Comment:
@SanketDG in the above comments @realslimshanky said it was not working for him for version 3.5.2, it would be great if you could update me and I can make the necessary changes. :)